### PR TITLE
Remove duplicate build and yarn install from lockfile

### PR DESCRIPTION
**Pull request by ChaituVR**
As suggested in https://github.com/snapshot-labs/lock/pull/38

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -8,26 +8,15 @@ on:
...
-    needs: build
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
           node-version: 14
           registry-url: https://registry.npmjs.org/
-      - run: yarn install
+      - run: yarn install --frozen-lockfile
```
Comment (mktcode) [inline .github/workflows/npm-publish.yml:19]: Now build is missing.

```suggestion
      - run: yarn install --frozen-lockfile
      - run: yarn build
```

Comment (ChaituVR) [reply .github/workflows/npm-publish.yml:19]: Ya ya, I know, ignored as we already have `prepublishOnly` script which build before publishing https://github.com/snapshot-labs/snapshot.js/blob/master/package.json#L53

Comment (mktcode) [reply .github/workflows/npm-publish.yml:19]: Does it?
https://github.com/yarnpkg/berry/issues/2305
https://yarnpkg.com/advanced/lifecycle-scripts

(Didn't know that too... 🤫)

Comment (ChaituVR) [reply .github/workflows/npm-publish.yml:19]: Yea with yarn we may have to change it to `prepack + prepublish` that's why we are still using `npm publish` 

which work fine in our current workflow
https://github.com/snapshot-labs/snapshot.js/runs/5079073430?check_suite_focus=true
image

Comment (mktcode) [reply .github/workflows/npm-publish.yml:19]: Ah ok cool! Got it.

**Review by mktcode — APPROVED** [review]

**Pull request merged**